packages/storybook/stories/segmented-control.mdx

```diff
@@ -18,13 +18,12 @@ A segmented control is a horizontal set of 2 to 5 exclusive segments, each fun
 ### When to use
 - Use when the change has an immediate effect on the UI [2].
 - Use when the options function as mutually exclusive buttons [2].
-- Use if your options are short and consistent [4].
 - Use when there is a default value [1].
 
 ### When not to use
 - Do not use in forms. Use checkboxes or radio buttons instead [2].
 - Do not use if you want to change a system state, use a toggle switch instead.
-- Do not use them to filter or navigate content, use Tabs instead [4].
+- Do not use to alternate between views within the same context, use Tabs instead [4].
```

Comment:
Si l'utilisateur alterne entre une vue en "tableau" et une vue en "chart", est-ce que tu considèrerais que ca "_alternate between views within the same context_"? Perso, je crois que c'est le genre d'utilisation que nous pourrions faire avec le composant mais la phrase ici semble indiquer le contraire. Ou ma perception de l'utilisation du composant n'est pas la bonne.